File: C#/RefactoringWorkshop/RefactoringWorkshop/Dispensable/DataClasses/CustomerSummaryView.cs
namespace RefactoringWorkshop.Dispensable.DataClasses
{
    public class CustomerSummaryView
    {
        private readonly Customer _customer;

        public CustomerSummaryView(Customer customer) {
            _customer = customer;
        }

        public string GetCustomerSummary() {
            var address = _customer.Address;
            return _customer.Title + " " + _customer.FirstName + " " + _customer.LastName + ", " +
                   address.City + ", " + address.PostCode + " " + address.Country;
        }
    }
}

File: Java/RefactoringWorkshop/src/main/java/dispensable/dataclasses/CustomerSummaryView.java
package dispensable.dataclasses;

public class CustomerSummaryView {

    private Customer customer;

    public CustomerSummaryView(Customer customer) {
        this.customer = customer;
    }

    public String getCustomerSummary() {
        Address address = customer.getAddress();
        return customer.getTitle() + " " + customer.getFirstName() + " " + customer.getLastName() + ", " +
                address.getCity() + ", " + address.getPostCode() + " " + address.getCountry();
    }
}
